### DISCUSSION TITLE
Unable to schedule the 3 Pods on 2 nodes both nodes are in different AZs

### DISCUSSION THREAD

**Discussion by mgup73** [strimzi/strimzi-kafka-operator#8896] [General]
Hi Team, 
Currently, I'm facing an issue in which my pod 1 of 3 broker/zookeeper is not getting scheduled in 2 nodes in different AZs.  Getting below error
0/2 nodes are available: 2 node(s) had volume node-affinity conflict. preemption: 0/2 nodes are available: 2 Preemption is not helpful for scheduling

Is this due to rack.topology that has been set to zones? How to resolve this issue?

**Comment by scholzj** [comment]
I think the error suggests that you have a Pod which uses a volume that cannot be used with any of the nodes. This could be for example because of using local storage or Amazon AWS EBS volumes which are tied to a particular AZ. But hard to say more without a detailed knowledge of your environment.

**Reply by mgup73** [reply]
Hi Yes, it's because volumes AZ are not matching. We used AZ a,b but the third volume uses c AZ. How to handle it?

**Reply by scholzj** [reply]
I think you need to look at your setup for how to handle it. As I said, without detailed knowledge of what your environment looks like it is hard to say exactly what the problem is. You might have misconfigured the Kafka cluster, your Kubernetes cluster might be badly set up etc.

**Reply by mgup73** [reply]
Found the issue volumes were in the wrong AZ. Thankyou :)
